articles/iot-hub/iot-hub-devguide-messages-read-builtin.md

```diff
@@ -75,5 +75,5 @@ If you want to route your device-to-cloud messages to custom endpoints, see [Use
 [lnk-d2c-tutorial]: tutorial-routing.md
 [lnk-event-hub-partitions]: ../event-hubs/event-hubs-features.md#partitions
 [lnk-servicebus-sdk]: https://www.nuget.org/packages/WindowsAzure.ServiceBus
-[lnk-eventprocessorhost]: http://blogs.msdn.com/b/servicebus/archive/2015/01/16/event-processor-host-best-practices-part-1.aspx
+[lnk-eventprocessorhost]: https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-dotnet-standard-getstarted-receive-eph
```

Comment:
Please remove the en-us element in the URL - it should be: https://docs.microsoft.com/azure/event-hubs/event-hubs-dotnet-standard-getstarted-receive-eph 
This means that people not in the en-us locale get automatically directed to a localized version of the page.